pkg/apis/imageregistry/v1/types.go

```diff
@@ -191,38 +191,17 @@ type ImageRegistryConfigStorageS3 struct {
 	CloudFront *ImageRegistryConfigStorageS3CloudFront `json:"cloudFront,omitempty"`
 }
 
-type ImageRegistryConfigStorageAzure struct {
-	Container string `json:"container,omitempty"`
-}
-
-type ImageRegistryConfigStorageGCS struct {
-	Bucket string `json:"bucket,omitempty"`
-}
-
-type ImageRegistryConfigStorageSwift struct {
-	AuthURL   string `json:"authURL,omitempty"`
-	Container string `json:"container,omitempty"`
-	Domain    string `json:"domain,omitempty"`
-	DomainID  string `json:"domainID,omitempty"`
-	Tenant    string `json:"tenant,omitempty"`
-	TenantID  string `json:"tenantID,omitempty"`
-}
```

Comment:
Since swift storage was added by a customer, we may mark this as experimental

Reply:
yeah, if there are users, it's better to have it as experimental rather than breaking them entirely

Reply:
@soltysh how to mark them as experimental?

Reply:
In the comments like here: https://github.com/openshift/api/blob/master/build/v1/types.go#L654-L659